lib/src/main/java/at/bitfire/vcard4android/contactrow/EventBuilder.kt

```diff
@@ -37,6 +37,7 @@ class EventBuilder(dataRowUri: Uri, rawContactId: Long?, contact: Contact, readO
     override fun build(): List<BatchOperation.CpoBuilder> {
         val result = LinkedList<BatchOperation.CpoBuilder>()
 
+        println("Contact birthday: ${contact.birthDay}")
```

Comment:
If we'd need logging, we could use `Constants.log`. But DAVx⁵ prints it anyway…

Reply:
Yeah, I think it shouldn't be there, I just forgot removing it